src/popup/popup.css

```
@@ -107,3 +107,98 @@ body {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.activity__recent li {
+  padding: 4px 8px;
+  font-size: 11px;
+  color: #4b5563;
+  border-left: 2px solid #e5e7eb;
+  margin-bottom: 2px;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)